src/lib/misc.c

```diff
@@ -602,12 +607,15 @@ pgp_memory_pad(pgp_memory_t *mem, size_t length)
 \param src Data to add
 \param length Length of data to add
 */
-void
+int
```

Comment:
hmmm... so pgp_memory_pad returns 1 in case of error and pgp_memory_add returns 0 in case of error. would be better if it is consistent